Makefile

```diff
@@ -132,6 +132,12 @@ build-streamlit-docker: ## Builds Docker with streamlit and models
 	aws s3 cp s3://datalabs-data/grants_tagger/models/label_binarizer.pkl models/
 	docker build -t streamlitapp -f Dockerfile.streamlit .
 
+.PHONY: install-private-requirements
+install-private-requirements: ## Install the private datascience utils
+	pip install pyodbc psycopg2
```

Comment:
Is that not part of the `setup.py` Of datascience

Reply:
No - and I need to fix this. But for now we have requirements.txt only.